.circleci/config.yml

```diff
@@ -650,15 +644,26 @@ workflows:
       - hold:
           type: approval
           requires:
+            - test
+            - assemble-purchase-tester
+            - run-backend-integration-tests
+            - run-revenuecatui-ui-tests
             - run-firebase-tests
             - run-firebase-tests-purchases-integration-test
             - run-firebase-tests-purchases-load-shedder-integration-test
             - run-firebase-tests-purchases-custom-entitlement-computation-integration-test
           <<: *release-branches
+      - force-release-without-waiting-for-tests:
+          type: approval
+          <<: *release-branches
       - revenuecat/tag-current-branch:
           requires:
             - hold
           <<: *release-branches
+      - revenuecat/tag-current-branch:
```

Comment:
Haven't checked, but I guess it's ok to have 2 of the same job in the same workflow but with different requirements?